### ISSUE TITLE
Space "start" time not updating correctly during large archive import

### ISSUE THREAD

**Issue by philrz** [brimdata/zed#2126]
Repro is on Brim `3334ea0` talking to `zqd` commit `47bb7c92` and testing with the "wrccdc year1" data set of Zeek logs.

Here I've imported the "wrccdc year1" Zeek logs into a Dev environment with the one change relative to Brim commmit `3334ea0` being to ensure that Archive Spaces are created when I import logs. The diff:

```
$ git diff
diff --git a/ppl/zqd/zqd.ts b/ppl/zqd/zqd.ts
index 769cfaa5..ec6cd078 100644
--- a/ppl/zqd/zqd.ts
+++ b/ppl/zqd/zqd.ts
@@ -158,6 +158,7 @@ export class ZQD {
       this.root,
       "-config",
       confFile,
+      "-filestorereadonly",
       "-suricatarunner",
       suricataRunnerCommand(this.suricataRunner),
       "-suricataupdater",
```

After I start the import and "lean back" and touch nothing, here's the end state after import completes:

![image](https://user-images.githubusercontent.com/5934157/107863327-664c4400-6e08-11eb-95f9-8049d72c3238.png)

However, if I select "Whole Space" from the time picker, the start time changes and a whole lot more events enter the picture, the histogram looks completely different, etc.

![image](https://user-images.githubusercontent.com/5934157/107863338-811eb880-6e08-11eb-838a-aae77eebedbb.png)

Here's how things look at the `zqd` layer:

```
$ zapi info zeek-logs
zeek-logs
  id:           sp_1oRfrvzqalwEqzflWVbD8kmXVEQ
  parent_id:    
  name:         zeek-logs
  data_path:    file:///Users/phil/work/brim/run/data/spaces/sp_1oRfrvzqalwEqzflWVbD8kmXVEQ
  storage_kind: archivestore
  span:         2017-03-24T17:14:32Z+30h50m57.340196001s
  size:         4.81GB
  pcap_support: false
  pcap_size:    0B
  pcap_path:    
```

Meanwhile, if I use an out-of-the-box nightly build based on the same Brim/zq commits (so, "Filestore" Spaces are made), I don't see the same symptom. Doing the same import and "lean back", the time span and histogram all look correct without having had to do any selection of "Whole Space".

![image](https://user-images.githubusercontent.com/5934157/107864597-9e588480-6e12-11eb-965c-385a9a3c198d.png)

The `zapi info` also looks exactly the same, with the only difference being the `storage_kind`.

```
$ zapi info zeek-logs
zeek-logs
  id:           sp_1oRiKAV2YsS0A9UHLMtxtfRNZMz
  parent_id:    
  name:         zeek-logs
  data_path:    file:///Users/phil/Library/Application%20Support/Brim/data/spaces/sp_1oRiKAV2YsS0A9UHLMtxtfRNZMz
  storage_kind: filestore
  span:         2017-03-24T17:14:32Z+30h50m57.340196001s
  size:         4.81GB
  pcap_support: false
  pcap_size:    0B
  pcap_path:    
```

While the symptom manifests in the app, I'm assuming the root cause of the problem is probably related to the incremental updates `zqd` sends back to the app about how the time span changes as the Space gets larger during import, hence opening this up on the zq side.

**Comment by philrz** [comment]
I had reason to check in on this one and I can see the primary issue seems to have gone away but another problem has taken its place. What I've observed is that starting with Brim commit `58f232c` (associated with brimdata/brim#1473) the originally-cited problem appeared to be fully fixed, with the histogram correctly rendered at the end of a "lean back" full import of the "year1" data set. Then once things advanced to the combination of Brim commit `89ecd27` pointing at Zed commit b8b93d7 (associated with #2548), the histogram is rendered correctly, but there's an "Empty log" pop-up that appears early during the import process that lingers until the user clicks **Dismiss**. A screenshot of this end state:

![](https://user-images.githubusercontent.com/5934157/116143714-536bb080-a690-11eb-8e43-8ae3b1b7537e.png)

I know we have further plans to change this area of frontend/backend communication as we complete the transition to Zed Lakes/Pools, and this area seems sensitive to breakage during changes, so I'll hold this one open and revisit it after the changes have settled down.

**Comment by philrz** [comment]
I've verified that with the full move to using Zed Lakes the symptom originally described now appears to be gone. Using nightly Brim build [https://storage.googleapis.com/brimsec-releases/brim/v0.25.0-prerelease-09efdcd9.0/macos/Brim-0.25.0-prerelease-09efdcd9.0.dmg](https://storage.googleapis.com/brimsec-releases/brim/v0.25.0-prerelease-09efdcd9.0/macos/Brim-0.25.0-prerelease-09efdcd9.0.dmg) (so, Brim commit `09efdcd` talking to Zed commit `709c769`) I imported the "wrccdc year1" Zeek logs and "leaned back" to let everything complete undisturbed. At the end, the histogram appeared to be drawn correctly, much as I was accustomed to seeing it in the days of Spaces based on File Stores. There were no excess error messages popped up either.

![](https://user-images.githubusercontent.com/5934157/116904893-26208480-abf3-11eb-8d4e-132f10fc2ca4.png)
